api/proto/index/v2/index.proto

```diff
@@ -0,0 +1,161 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+package index.v2;
+option go_package = "./index/v2;index";
+
+// Service provides an external API for managing the index records
```

Comment:
The description is not relevant, proposal: "Service provides an API for working with the index objects (nodes, records, etc)". The assumption is that Simila is the index as a whole built with a certain structure and objects in mind.

Reply:
ok